webrtc-stats.html

```diff
@@ -976,7 +976,8 @@
              unsigned long long bytesSent;
              double             targetBitrate;
              unsigned long      framesEncoded;
-};</pre>
+             double             totalEncodeTime;
+          };</pre>
```

Comment:
(I thought the }; not being aligned was a mistake but I find non-indented }; all over the place? Should I not fix the indentation?)